feature_tests/dart/lib/src/BorrowedFields.g.dart

```diff
@@ -75,6 +81,6 @@ final class BorrowedFields {
 }
 
 @meta.ResourceIdentifier('BorrowedFields_from_bar_and_strings')
-@ffi.Native<_BorrowedFieldsFfi Function(ffi.Pointer<ffi.Opaque>, ffi.Pointer<ffi.Uint16>, ffi.Size, ffi.Pointer<ffi.Uint8>, ffi.Size)>(isLeaf: true, symbol: 'BorrowedFields_from_bar_and_strings')
+@ffi.Native<_BorrowedFieldsFfi Function(ffi.Pointer<ffi.Opaque>)>(isLeaf: true, symbol: 'BorrowedFields_from_bar_and_strings')
```

Comment:
issue: the arguments disappeared

Reply:
Fixed. The `param_types_ffi_cast` etc stuff is kinda confusing, we generate a lot of different things in one function. I didn't bother to try and understand it too deeply but we should document this better if we get the chance.

(Also now that this is no longer doing two-parameter stuff we probably can clean things up)